docs/scos/dev/tutorials-and-howtos/introduction-tutorials/tutorial-sending-an-email.md

```diff
@@ -126,231 +121,133 @@ Create the `MailType` plugin for this example. For more information about creati
-
-        return $this;
+        $customerTransfer = $mailTransfer->requireCustomer()->getCustomer();
+    
+        return $mailTransfer
+            ->setSubject(static::GLOSSARY_KEY_MAIL_SUBJECT)
+            ->addTemplate(
+                (new MailTemplateTransfer())
+                    ->setName(static::MAIL_TEMPLATE_HTML)
+                    ->setIsHtml(true),
+            )
+            ->addTemplate(
+                (new MailTemplateTransfer())
+                    ->setName(static::MAIL_TEMPLATE_TEXT)
+                    ->setIsHtml(false),
+            )
+            ->addRecipient(
+                (new MailRecipientTransfer())
+                    ->setEmail($customerTransfer->getEmail())
+                    ->setName(sprintf('%s %s', $customerTransfer->getFirstName(), $customerTransfer->getLastName())),
+            );
     }
 }
 ```
-
 </details>
 
-The `Mail` module’s default `MailBuilder` is already pre-defined to build the `MailTransfer`. `MailBuilder` internally adds a new `MailRecipientTransfer` with the passed information, email, and name.
-
 ## 3. Registering a plugin
 
 When the plugin is created, it must be registered in `MailDependencyProvider`:
 
 ```php
 <?php
-        ...
-        $container->extend(self::MAIL_TYPE_COLLECTION, function (MailTypeCollectionAddInterface $mailCollection) {
-            ....
-            $mailCollection->add(new CustomCustomerRegistrationMailTypePlugin());
-
-            return $mailCollection
-        });
-        ...
- ?>
-```
-
-## 4. Mail translations
-
-The default `MailBuilder` also has access to the glossary with the `setSubject()` method. This is used for translations as follows:
-
-```php
-<?php
-namespace Pyz\Zed\Customer\Communication\Plugin\Mail;
+namespace Pyz\Zed\Mail;
 
-use Spryker\Zed\Kernel\Communication\AbstractPlugin;
-use Spryker\Zed\Mail\Business\Model\Mail\Builder\MailBuilderInterface;
-use Spryker\Zed\Mail\Dependency\Plugin\MailTypeInterface;
+use Pyz\Zed\Customer\Communication\Plugin\Mail\CustomCustomerRegistrationMailTypeBuilderPlugin;
+use Spryker\Zed\Mail\MailDependencyProvider as SprykerMailDependencyProvider;
 
-class CustomerCustomerRegistrationMailTypePlugin extends AbstractPlugin implements MailTypePluginInterface
+class MailDependencyProvider extends SprykerMailDependencyProvider
 {
-    ...
-    protected function setSubject(MailBuilderInterface $mailBuilder)
+    protected function getMailTypeBuilderPlugins(): array
     {
-        $mailBuilder->setSubject('mail.customer.registration.subject');
+        return [
+            new CustomCustomerRegistrationMailTypeBuilderPlugin(),
+        ];
     }
-    ...
 }
 ```
 
-A string is used as a key of the translation. The `MailBuilder` internally does the translation through `GlossaryFacade`:
+## 4. Mail translations
+
+The `MailTypeBuilderPlugin` also has access to the glossary with the `setSubject()` method. This is used for translations as follows:
```

Comment:
1.  `subject` is not the only field that can be translated, please adjust the description.
2. Please adjust an example to show how to translate with params.

Reply:
1. Added the note section
2. It was there, in the section below, or you mean something else?

Reply:
Example (As well as description), should include 
```
| MailRecipient.nameTranslationParameters | property | added | src/Generated/Shared/Transfer/MailRecipientTransfer |
| MailSender.nameTranslationParameters | property | added | src/Generated/Shared/Transfer/MailSenderTransfer |
```